camel/agents/chat_agent.py

```diff
@@ -196,6 +196,7 @@ def __init__(
         self.func_dict = {
             tool.get_function_name(): tool.func for tool in all_tools
```

Comment:
We may not need `self.func_dict` anymore after adding `__call__` in `FunctionTool`. Can @Wendong-Fan or @WHALEEYE help with checking and fixing it?